steps/identity/appointee-form-download/AppointeeFormDownload.js

```diff
@@ -1,19 +1,30 @@
 'use strict';
 
-const { ExitPoint } = require('@hmcts/one-per-page');
+const { form, textField } = require('@hmcts/one-per-page/forms');
+const { Question } = require('@hmcts/one-per-page');
 const paths = require('paths');
 
-class AppointeeFormDownload extends ExitPoint {
+class AppointeeFormDownload extends Question {
 
     static get path() {
 
         return paths.identity.downloadAppointeeForm;
     }
 
-    get form() {}
+    get benefitType() {
+        return this.fields.benefitType.value;
+    }
+
+    get form() {
 
+        return form(
+
+            textField.ref(this.journey.steps.BenefitType, 'benefitType')
+        );
+    }
 
     next() {}
```

Comment:
There's no need to defined next() in this instance, you can safely remove it.

Reply:
done